metrics.py

```diff
@@ -464,6 +469,74 @@ def ingest_dashboard_revision_get():
 
     return None
 
+def ingest_attributes(api):
```

Comment:
Would seem better to order after `iingest_dashboard()` since all the map functions above are related and named as such.